client/lib/invites/notices.js

```diff
@@ -7,51 +7,131 @@ import get from 'lodash/object/get'
 
 const InviteNotices = {};
 
+function displayNotice( invite, displayOnNextPage ) {
+	switch ( get( invite, 'role' ) ) {
+		case 'follower':
+			notices.success(
+				i18n.translate(
+					'You are now following %(site)s', {
+						args: { site: get( invite, 'site.title' ) }
+					}
+				),
+				{
+					button: i18n.translate( 'Visit Site' ),
+					href: get( invite, 'site.URL' ),
+					displayOnNextPage
+				}
+			);
+			break;
+		case 'administrator':
+			notices.success(
+				<div>
+					<h3 className="invite-message__title">
+						{ i18n.translate( 'You\'re now an Administrator of: %(site)s', {
+							args: { site: get( invite, 'site.title' ) }
+						} ) }
+					</h3>
+					<p className="invite-message__intro">
+						{ i18n.translate( 'This is your site dashboard where you can write posts and control your site. ' ) }
+					</p>
+					<p className="invite-message__intro">
+						{
+							i18n.translate(
+								'Since you\'re new, you might like to {{docsLink}}take a tour{{/docsLink}}.',
+								{ components: { docsLink: <a href="http://en.support.wordpress.com/" target="_blank" /> } }
+							)
+						}
+					</p>
+				</div>,
+				{ displayOnNextPage }
+			);
+			break;
+		case 'editor':
+			notices.success(
+				<div>
+					<h3 className="invite-message__title">
+						{ i18n.translate( 'You\'re now an Editor of: %(site)s', {
+							args: { site: get( invite, 'site.title' ) }
+						} ) }
+					</h3>
+					<p className="invite-message__intro">
+						{ i18n.translate( 'This is your site dashboard where you can write posts and control your site. ' ) }
```

Comment:
I'm not sure that an editor can "control a site". The [Codex](https://codex.wordpress.org/Roles_and_Capabilities) describes an editor as:

> somebody who can publish and manage posts including the posts of other users.

Here's an example of what we use in [invite-form-header](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/invites/invite-form-header/index.jsx#L207):

> As an editor, you will be able to publish and manage your own posts and the posts of others, as well as upload media.